custom_reports/http_api/definitions/query_request.yaml

```diff
@@ -99,6 +98,69 @@ properties:
     required:
       - operator
       - conditions
+  metric_filters:
+    type: object
+    description: |
+      Metric filters for the query. The top-level object must specify AND/OR
+      clause, but any nested objects may either specify another AND/OR
+      clause or a single filter.
+    properties:
+      operator:
+        type: string
+        description: Logical operator (AND/OR) for the clause.
+        enum:
+          - and
+          - or
+      conditions:
+        type: array
+        description: List of filters or recursively nested clauses.
+        items:
+          type: object
+          required:
+            - column_id
+            - condition
+          properties:
+            column_id:
+              type: string
+              description: |
+                ID of the column (must be a metric, or a dimension converted
```

Comment:
```suggestion
                ID of the column (must be a metric, or a column converted
```

Reply:
Why?

Reply:
I think we also have metric -> metric conversions. This would be more general description. We can leave previous one if we don't have metric -> dimension conversions.

Reply:
Transformations by definition may only be applied to dimensions (i.e. the raw data).